services/worker-manager/test/worker_scanner_test.js

```diff
@@ -117,16 +122,18 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
         workerId: "testing-123",
       });
       assert(worker1.providerData.checked);
+      assert(worker1.expires > expires);
```

Comment:
here a comment could be useful that the expiry should be updated since expiry was less than a day